vaadin-rich-text-editor-flow-parent/vaadin-rich-text-editor-flow-integration-tests/src/main/java/com/vaadin/flow/component/richtexteditor/tests/RichTextEditorAsDeltaPage.java

```diff
@@ -0,0 +1,105 @@
+package com.vaadin.flow.component.richtexteditor.tests;
+
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.component.richtexteditor.RichTextEditor;
+import com.vaadin.flow.data.binder.Binder;
+import com.vaadin.flow.data.binder.BinderValidationStatus;
+import com.vaadin.flow.data.binder.BindingValidationStatus;
+import com.vaadin.flow.data.value.ValueChangeMode;
+import com.vaadin.flow.function.SerializablePredicate;
+import com.vaadin.flow.router.Route;
+
+import java.io.Serializable;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Route(value = "vaadin-rich-text-editor/as-delta")
+public class RichTextEditorAsDeltaPage extends VerticalLayout {
```

Comment:
The IT setup is adapted from the existing ITs testing the value property (`MainView` / `BasicUseIT`) with some cleanup to make them more readable.